src/App.js

```diff
@@ -1,6 +1,10 @@
 import React, {useEffect} from 'react';
 import './App.css'; 
 import axios from 'axios'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
```

Comment:
I might put some white space between external components and internal components